### PR TITLE
Updating apko docs

### PR DESCRIPTION
**Pull request by erikaheidi**
This PR brings a few updates to the apko documentation in Academy, simplifying some things and adding more relevant info in the FAQ section.

The reference page was removed since we don't have the bandwidth to keep it up-to-date. I added an Nginx redirect to redirect requests to this page to the [apko file format](https://github.com/chainguard-dev/apko/blob/main/docs/apko_file.md) in the official repo.

### CODE REVIEW

File: content/open-source/build-tools/apko/faq.md [hunk 1]
```diff
@@ -18,20 +18,19 @@ toc: true
 ---
 
 ## Do I need to understand apko to use Chainguard Images?
-No. Chainguard built [apko](https://github.com/chainguard-dev/apko) as part of its open source tooling under the [Wolfi](/open-source/wolfi) operating system. While you can check out the [project on GitHub](https://github.com/chainguard-dev/apko) and learn more, it's not a prerequisite for using or [working with Chainguard Images](/chainguard/chainguard-images/working-with-images).
+No. Chainguard built [apko](https://github.com/chainguard-dev/apko) as part of its open source tooling around the [Wolfi](/open-source/wolfi) operating system. While you can check out the [project on GitHub](https://github.com/chainguard-dev/apko) and learn more, it's not a prerequisite for using or [working with Chainguard Images](/chainguard/chainguard-images/working-with-images).
 
-## How are melange packages defined?
-melange apks are defined declaratively using a YAML file.
+## How are apko images defined?
+apko images are defined declaratively using a YAML file. It was designed this way to facilitate reproducible builds - run apko twice, and you'll get the same output.
```
Comment (SharpRake) [inline content/open-source/build-tools/apko/faq.md:24]:
```suggestion
apko images are defined declaratively using a YAML file. It was designed this way to facilitate reproducible builds — run apko twice, and you'll get the same output.
```

File: content/open-source/build-tools/apko/faq.md [hunk 2]
```diff
@@ -18,20 +18,19 @@ toc: true
...
 
-## Are melange apks compatible with Alpine?
-Yes, melange apks are built to be compatible with apk-based systems including Alpine.
+## Does apko provide SBOMs?
+Yes, apko builds include high-quality SBOMs (software bills of materials) for all builds. This is a key feature of the tooling that Chainguard has developed to ensure that users can trust the software they are running.
```
Comment (SharpRake) [inline content/open-source/build-tools/apko/faq.md:27]: nbd but you could link to the SBOM explainer (https://edu.chainguard.dev/open-source/sbom/what-is-an-sbom/) if you want

**Review by SharpRake — APPROVED** [review]
looks good to me!

File: content/open-source/build-tools/apko/faq.md (hunk 2 above)
Comment (erikaheidi) [reply content/open-source/build-tools/apko/faq.md:27]: good idea!

**Pull request merged**